pyuo/brain.py

```diff
@@ -54,8 +54,14 @@ def run(self):
 		''' This is the main Brain thread entry point, contains the main loop, internal '''
 
 		self.init()
-
-		main = threading.main_thread()
+		
+		try:
+			main = threading.main_thread()
+		except:
```

Comment:
Is there a reason for catching any exception? You meant to catch AttributeError?